pkgs/development/tools/language-servers/postgres-lsp/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, protobuf
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "postgres-lsp";
+  version = "unstable-2023-08-08";
+
+  src = fetchFromGitHub {
+    owner = "supabase";
+    repo = "postgres_lsp";
+    rev = "1250f5ed14a0e86b2b7fa581214284c67b960621";
+    hash = "sha256-Y43sTgKNcAI3h6McDc0g6o9CX6jOKBfURLWyjJhvmwk=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+  };
+
+  postPatch = ''
+    ln -s ${./Cargo.lock} Cargo.lock
+  '';
```

Comment:
Can we add an issue upstream that they should add the `Cargo.lock` file to their repository?

Reply:
If yes, the issue should be put in a comment here.

Reply:
created a pull request and added a comment